Dockerfile

```diff
@@ -16,6 +16,9 @@ LABEL name="openshift-ansible" \
 
 USER root
 
+# Create a symlink to /opt/app-root/src so that files under /usr/share/ansible are accessible
+RUN mkdir -p /usr/share/ansible/ && ln -s /opt/app-root/src /usr/share/ansible/openshift-ansible
+
```

Comment:
Now that we understand better the need for this, could we expand the comment a bit more? e.g. mention that the default `PLAYBOOK_FILE` in `config.json.template` must use a full path based on the RPM-based installation, and therefore this symlink is needed for this to work...